hedy.py

```diff
@@ -39,14 +39,15 @@
     'nl': 'Nederlands',
     'es': 'Español',
     'fr': 'Français',
+    'nb_no': 'Norsk',
```

Comment:
'Norsk Bokmål' please 🧡

Reply:
We have two written languages in Norway (technically three if you count the old native language Samii)